### ISSUE TITLE
 Latest version 1.3.3 needs a go mod tidy

### ISSUE THREAD

**Issue by gindelmer** [mgechev/revive#894]
**Describe the bug**
 My run of dependabot tripped on the version 1.3.3 of this repo:
```
go: github.com/mgechev/revive@latest: github.com/mgechev/revive@v1.3.3: verifying module: checksum mismatch
	downloaded: h1:H5KpQYVmht69pg47kbIg/E4S1jA1yfFBEwy7Zv3uDX8=
	sum.golang.org: h1:GUWzV3g185agbHN4ZdaQvR6zrLVYTUSA2ktvIinivK0=
SECURITY ERROR
```

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
# flags

revive ...
```

```toml
# config file
```

**Expected behavior**

**Logs**

**Desktop (please complete the following information):**
 - OS: Debian Bullseye
 - Version of Go: 1.19

**Additional context**

**Comment by denisvmedia** [comment]
1.3.3 should be compiled with go 1.20 or 1.21. 1.19 is not maintained anymore. Can you try doing that with 1.20 or 1.21?

**Comment by gindelmer** [comment]
I think the problem comes with an  overwrite of the tag 1.3.3 like in https://github.com/mgechev/revive/issues/632
I updated the golang in my pipelines to 1.20.8 and the problem stays.

```
go: github.com/mgechev/revive@latest: github.com/mgechev/revive@v1.3.3: verifying module: checksum mismatch
	downloaded: h1:H5KpQYVmht69pg47kbIg/E4S1jA1yfFBEwy7Zv3uDX8=
	sum.golang.org: h1:GUWzV3g185agbHN4ZdaQvR6zrLVYTUSA2ktvIinivK0=
SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

**Comment by chavacava** [comment]
Hi @gindelmer thanks for creating the issue. The v1.3.4 should remove the problem.

**Comment by gindelmer** [comment]
Many thanks, problem is fixed with v1.3.4.
